src/tests/ftest/nvme/enospace.yaml

```diff
@@ -0,0 +1,62 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-C
+    - client-D
+#Few tests are in loop for ~10 times and single loop
+#is taking ~600 seconds to finish so larger timeout to run tests.
+timeout: 7500
+server_config:
+  servers_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      bdev_class: nvme
+      bdev_list: ["0000:81:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+      targets: 1
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      bdev_class: nvme
+      bdev_list: ["0000:da:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+      targets: 1
+pool:
+    mode: 146
+    name: daos_server
+    scm_size: 5368709120 #5G
+    nvme_size: 5368709120 #5G
+    control_method: dmg
+container:
+    control_method: daos
+ior:
+  api: "DAOS"

```

Comment:
Please use `DFS` driver instead of `DAOS`. `DAOS` driver will soon be deprecated